corehq/apps/users/templates/users/partials/linked_roles_modal.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+
+<div id="modal_linked_roles" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+        <div class="modal-header">
+            <button type="button"
+                    class="close" data-dismiss="modal">
+            <span aria-hidden="true">&times;</span>
+            </button>
+            <h3 class="modal-title">
+                <i class="fa fa-link"></i>
+                Unlock Linked Roles?
+            </h3>
+        </div>
+        <div class="modal-body">
+            <p>Linked Roles are roles that are controlled by a separate project space linked to this project.
+                Click here to learn more.</p>
+            <p>Changes made to Linked Roles may be overwritten by the next updates from the linked project space,
+                and may also affect solutions on this project space.</p>
```

Comment:
I find the wording "may also affect solutions on this project space" a bit strange. What is meant by "solutions" here?